LSP-metals.sublime-commands

```diff
@@ -31,5 +31,6 @@
             "base_file": "${packages}/LSP-metals/LSP-metals.sublime-settings",
             "default": "{\n\t$0\n}\n"
         }
-    }
+    },
+    { "caption": "LSP-metals: New Java File", "command": "lsp_metals_new_java_file", "args":{"paths": []}},
```

Comment:
I believe you can do something like this without creating `LspMetalsNewJavaFileCommand`.
```suggestion
    { "caption": "LSP-metals: New Java File", "command": "lsp_metals_text_command", "args":{"command_name": "new-java-file", "command_args": ["${file}"]}},
```

Reply:
I didn't know about this macro :) but it doesn't work because the path doesn't include the schema which expected by Metals. This why the path in wrapped by `filename_to_uri`  in `lsp_metals_new_java_file.py`

Are these macros documented somewhere ?

Reply:
Can use `${file_uri}` then.